packages/pwa-kit-react-sdk/src/ssr/universal/components/redirect-with-status/index.test.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+import {render} from '@testing-library/react'
+import {StaticRouter, Route} from 'react-router-dom'
+import {RedirectWithStatus} from './index'
+
+describe('RedirectWithStatus', () => {
+    // Dummy test needed for code coverage (the case where there is no static context)
```

Comment:
For this test case, there is actually a static context (and it's empty). You can add a console log to verify that your if-condition actually gets run.

How about converting this test to verify that without passing in a status code, by default the context will be set with 302 status?

Reply:
I've updated the test to verify that a redirect still happens if there is no context.